cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -451,7 +428,7 @@ public Secret generateBrokersClientsSecret() {
         if (rack != null) {
             volumeMountList.add(createVolumeMount(RACK_VOLUME_NAME, RACK_VOLUME_MOUNT));
         }
-        volumeMountList.add(createVolumeMount("internal-certs", "/opt/kafka/internal-certs"));
+        volumeMountList.add(createVolumeMount("cluster-certs", "/opt/kafka/cluster-certs"));
         volumeMountList.add(createVolumeMount("clients-certs", "/opt/kafka/clients-certs"));
```

Comment:
ditto as above singular "clients-cert". 
Out of curiosity, are you using a different folder for putting this cert for security reason I guess.

Reply:
You mean separate folder for the clients CA certificate and for the broker certificates? They are from separate secrets, so we avoid any possible naming collisions for the future. I guess I would also need to mount it file by file in order to not overwrite the folder (but maybe I'm wrong about this one).